pygcdl.py

```diff
@@ -404,10 +425,12 @@ def _format_dsvars(self, dsvars):
             dsvars = pd.DataFrame(dsvars, columns=["dataset", "variable"])
             dsvars = dsvars.iloc[:,0:2]
         elif isinstance(dsvars, dict):
-            dsvars = pd.DataFrame(columns=["dataset", "variable"])
+            dsvars_out = pd.DataFrame(columns=["dataset", "variable"])
             for dataset, var_list in dsvars.items():
                 for var in var_list:
-                    dsvars.loc[len(dsvars.index)] = [dataset, var]
+                    dsvars_out.loc[len(dsvars_out.index)] = [dataset, var]
+            dsvars = dsvars_out
+            print("dsvars_out: ", dsvars_out)

```

Comment:
Is this print statement intentional or leftover from debugging?

Reply:
Fixed on fork